.circleci/config.yml

```diff
@@ -1,17 +1,16 @@
 version: 2
 
-xcode_version: &xcode_version 9.4
+xcode_version: &xcode_version 10.0
 iphone_name: &iphone_name iPhone 8
-ios_version: &ios_version 11.4
+ios_version: &ios_version 12.0
```

Comment:
Ok, the issue here is that the environment variable `IOS_VERSION` is being interpolated as `12` in the `Makefile` when it executes the command to run tests. This needs to be `12.0`.

We actually only use this env var in the `Makefile` in one place so you have two options:

1. Remove this line from the `.circleci/config.yml` and the reference to it below on line `23`. This will cause it to use the default value set in the `Makefile` and it'll work.
or
2. Try changing this value in the `.circleci/config.yml` to be "12.0" although I think that would cause it to print out in the command as `="12.0"` and that would fail too.